apps/YellowOnBoardActuators/components/outputActuator/outputActuator.h

```diff
@@ -1,6 +1,28 @@
+//--------------------------------------------------------------------------------------------------
+/**
+ * Boolean output actuator utility.  Does the bulk of the work to register a Boolean actuator in
+ * the Data Hub and translates updates from the Data Hub into a '0' or '1' written to a sysfs file.
+ *
+ * Copyright (C) Sierra Wireless Inc.
+ */
+//--------------------------------------------------------------------------------------------------
 #ifndef _OUTPUT_ACTUATOR_H_
 #define _OUTPUT_ACTUATOR_H_
 
-LE_SHARED le_result_t RegisterOutputActuator(const char *dhubBasePath, const char *sysfsPath);
+//--------------------------------------------------------------------------------------------------
+/**
+ * Register a Boolean actuator, making it appear at a given resource path in the Data Hub and
```

Comment:
Well, then I would argue that the .h should be documented better than the .c, because the .h is the interface (contract) with the caller, and is free from the implementation details.
But, it's just a copy and paste, so I just generally put the docs in both places.